HISTORY.md

```diff
@@ -1,5 +1,12 @@
 # Version History
 
+## 0.0.27-preview / 2021-05-24
```

Comment:
i was incrementing the minor version of the sample since the functionality went from 12 to 14 steps. the python library itself to handle the new properties i only incremented the build number there as well.

Reply:
I could go either way, but this felt more build number-y to me

Reply:
I suppose that's true, we incremented the minor version for the assets changes but not the equivalent of these changes. I'm OK with it as-is.